HACKING/.env.template

```diff
@@ -0,0 +1,67 @@
+# Proxstar
+PROXSTAR_VM_EXPIRE_MONTHS=3
+PROXSTAR_VNC_CLEANUP_TOKEN=
+
+# Development options
+# Determines weather or not to run STARRS queries (for doing stuff like checking for available IPs)
+PROXSTAR_USE_STARRS=True
+# If you're an RTP and want to see a normal user's homepage view, set this to True.
+PROXSTAR_FORCE_STANDARD_USER=False
+
+# Flask
+PROXSTAR_IP=0.0.0.0 # The IP address to which Proxstar is served.
+PROXSTAR_PORT=8000 # The port Proxstar runs on.
+PROXSTAR_SERVER_NAME=localhost:8000 # It's configured this way in SSO.
+PROXSTAR_SECRET_KEY= # This can be LITERALLY anything.
+
+# OIDC
+PROXSTAR_OIDC_ISSUER=https://sso.csh.rit.edu/auth/realms/csh
+PROXSTAR_CLIENT_ID= # Ask an RTP. Can get this from OKD configs.
+PROXSTAR_CLIENT_SECRET= # Ask an RTP.
+PROXSTAR_REDIRECT_URI=https://proxstar.csh.rit.edu/logout
+
+# Proxmox
+PROXSTAR_PROXMOX_HOSTS= # Host list that Proxstar can use (proxstar01, etc...)
+PROXSTAR_PROXMOX_USER=api@pve
+PROXSTAR_PROXMOX_PASS= # Password for proxstar user
+PROXSTAR_PROXMOX_ISO_STORAGE=nfs-iso
+PROXSTAR_PROXMOX_VM_STORAGE=ceph
+PROXSTAR_PROXMOX_SSH_USER=root
+PROXSTAR_PROXMOX_SSH_KEY="" # Ask an RTP. This is gonna look like a certificate.
+PROXSTAR_PROXMOX_SSH_KEY_PASS= # Password for above certificate
+
+# STARRS
+PROXSTAR_STARRS_DB_HOST=proxstar-postgres
+PROXSTAR_DB_NAME=starrs
+PROXSTAR_DB_USER=postgres
+PROXSTAR_DB_PASS=tits12347
```

Comment:
Please don't publish this is in public open source repo

Reply:
These are credentials for the development DB container.

Reply:
Then change the password